rules/integrations/aws/persistence_route_53_hosted_zone_associated_with_a_vpc.toml

```
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2021/07/19"
+maturity = "production"
+updated_date = "2021/07/19"
+
+[rule]
+author = ["Austin Songer"]
+description = "Identifies when a Route53 hosted zone has been associated with VPC."
+false_positives = [
+    """
+    A hosted zone may be asssociated with a VPC by a system or network administrator. Verify whether the user
+    identity, user agent, and/or hostname should be making changes in your environment. If known behavior is causing false positives, it can be exempted from the
+    rule.
+    """,
+]
+from = "now-60m"
+index = ["filebeat-*", "logs-aws*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS Route53 hosted zone associated with a VPC"
```

```suggestion
name = "AWS Route53 private hosted zone associated with a VPC"
```
From what I got, this is limited to private hosted zones